src/routes/dashboard/helpers/channels.ts

```diff
@@ -0,0 +1,75 @@
+import type { Question, Questions } from '../types'
+
+export function sortChannels(questions: Question[]) {
+  const counts = questions.reduce((count, question) => {
+    const key = question.channelName
+    return count[key] ? ++count[key] : (count[key] = 1), count
+  }, {})
+
+  return Object.entries(counts)
+    .map(([channelName, count]) => {
+      return { group: channelName, count }
+    })
+    .sort((a, b) => a.group.localeCompare(b.group))
+}
+
+const percentColors = [
+  { pct: 0, color: { r: 0xff, g: 0x00, b: 0 } },
+  { pct: 50, color: { r: 0xff, g: 0xff, b: 0 } },
+  { pct: 100, color: { r: 0x00, g: 0xff, b: 0 } },
+]
+
+const getColor = (pct: number) => {
```

Comment:
minor nit, can we use `percent` instead of `pct`? this has me wanting to go hiking 🏕